apps/transport/lib/transport/consolidated_dataset.ex

```diff
@@ -0,0 +1,85 @@
+defmodule Transport.ConsolidatedDataset do
+  @moduledoc """
+  A module to hold configuration of the datasets used in geodata queries and access them easily.
+  """
+
+  import Ecto.Query
+
+  @config %{
+    irve: %{dataset_type: "charging-stations", publisher: :datagouvfr},
+    bnlc: %{dataset_type: "carpooling-areas", publisher: :transport},
+    parkings_relais: %{dataset_type: "private-parking", publisher: :transport},
+    zfe: %{dataset_type: "low-emission-zones", publisher: :transport}
```

Comment:
```suggestion
    irve: %{dataset_type: "charging-stations", publisher: :datagouvfr},
    bnlc: %{dataset_type: "carpooling-areas", publisher: :transport},
    parkings_relais: %{dataset_type: "private-parking", publisher: :transport},
    zfe: %{dataset_type: "low-emission-zones", publisher: :transport}
```
Utiliser `publisher_id: publisher_id(…)` directement ici ?

Reply:
Non, un poil ennuyant pour des raisons de ce qui est accessible à la compilation et tout.